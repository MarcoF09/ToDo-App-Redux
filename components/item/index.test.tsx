import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'
import { Item } from './index'

describe('should render button', () => {
  it('renders correctly', () => {
    const component = shallow(
      <Item
        title="Title"
        description="Description"
        completed={true}
        onPress={jest.fn()}
        checkboxClick={jest.fn()}
      />
    )
    expect(toJson(component)).toMatchSnapshot()
  })
})
