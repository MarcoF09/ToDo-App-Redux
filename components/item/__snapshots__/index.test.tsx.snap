// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render button renders correctly 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[MockFunction]}
  style={
    Object {
      "backgroundColor": "white",
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "#f0f0f0",
        "borderWidth": 1,
        "flex": 1,
        "flexDirection": "row",
        "paddingBottom": 16,
        "paddingLeft": 30,
        "paddingRight": 16,
        "paddingTop": 16,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontFamily": "SourceSansPro-Regular",
            "fontSize": 16,
            "lineHeight": 24,
          }
        }
      >
        Title
      </Text>
      <Text
        style={
          Object {
            "color": "#959595",
            "fontFamily": "SourceSansPro-Regular",
            "fontSize": 14,
            "lineHeight": 20,
          }
        }
      >
        Description
      </Text>
    </View>
    <CheckBox
      checked={true}
      click={[MockFunction]}
    />
  </View>
</TouchableOpacity>
`;
